libs/common/storybook/src/stories.helper.ts

```diff
@@ -76,11 +76,11 @@ export const storyArgs = (args: Args) => {
     };
 };
 
-export const storyArgTypes = (argTypes: ArgTypes) => {
+export const storyArgTypes = (argTypes: ArgTypes, next = false) => {
     return {
         ...argTypes,
         customCSS: {
-            name: 'data-vl-custom-css',
+            name: !next ? 'data-vl-custom-css' : 'custom-css',
```

Comment:
Waar dient dit ! voor?

Reply:
om de boolean te inverten; had ooit eens gelezen dat bij het verwerken van code de default conditie altijd standaard berekend wordt en je daarom beter voor performance redenen, de meest voorkomende conditie in de `true` conditie moet plaatsen;

ik doe dit sindsdien ook altijd bij de meeste if than else condities; maar even gegoogled en dit blijkt eerder enkel in low level code environments